cinema/views.py

```diff
@@ -31,9 +44,13 @@ class CinemaHallViewSet(viewsets.ModelViewSet):
 
 
 class MovieViewSet(viewsets.ModelViewSet):
-    queryset = Movie.objects.all()
+    queryset = Movie.objects.prefetch_related("genres", "actors")
```

Comment:
maybe i'm wrong but its redundant to override queryset variable here, it neccessary only in get_queryset method. Or u can override it here and don't in get_queryset method